# Limiar de definição entre branco e preto utilizado pela função treshould do openCV
int64 limiarDefinicao

